cloud/aws/bin/dumpdb.py

```diff
@@ -0,0 +1,142 @@
+import os
+import sys
+import tempfile
+import shlex
+import subprocess
+from time import sleep
+import urllib.request
+
+from cloud.aws.templates.aws_oidc.bin import resources
+from cloud.aws.templates.aws_oidc.bin.aws_cli import AwsCli
+from cloud.shared.bin.lib import terraform
+from cloud.shared.bin.lib.config_loader import ConfigLoader
+from cloud.shared.bin.lib.print import print
+
+
+def run(config: ConfigLoader):
+    aws = AwsCli(config)
+    print(
+        "\n"
+        "!!! WARNING !!!: This command will create a dump of the entire database, including personally identifiable information (PII). Ensure you take the utmost care in handling this data and store it in a secure location.\n"
+        "\n"
+        'This process will set up a temporary EC2 host with access to the database, use SSH to run the pg_dump command on that host, then SCP the file to this machine. '
+        'You will need to confirm the application of the Terraform manifest that creates these temporary resources, and then confirm the teardown of these resources. '
+        'If something goes wrong and this process is interrupted before it tears down the resources, you can find them all with the "Module = dbaccess" tag in the AWS console. They should be deleted manually.'
+        "\n")
+    # Current working dir should be the 'checkout' folder, so go one level above.
+    default_file = os.path.join(
+        os.path.dirname(os.getcwd()), 'civiform_database.dump')
```

Comment:
**suggestion:**  I think its worth including the var.app_prefix in this filename. I can imagine if someone has multiple instances they may get confused between different dumps. You can also consider including the date.